vertical-pod-autoscaler/deploy/vpa-v1-crd-gen.yaml

```diff
@@ -383,6 +383,12 @@ spec:
                   pods. If not specified, all fields in the `PodUpdatePolicy` are
                   set to their default values.
                 properties:
+                  minReplicas:
+                    description: Overrides global '--min-replicas' flag, specifying
```

Comment:
I'd change the order of information here.

Minimal number of replicas needed for Updater to attempt pod eviction for pods controlled by this VPA object. Overrides `--min-replicas` flag.

I expect some users of VPA API won't know about VPA flags.

Reply:
Done.